packages/ui/src/lib/adapters/zkitter/index.ts

```diff
@@ -588,8 +562,24 @@ export class ZkitterAdapter implements Adapter {
 			ecdsa,
 		}))
 
-		await this.syncChats()
+		const contract = getGlobalAnonymousFeed()
+
+		const { IndexedDBConnector } = await import('anondb/web');
+
+		const db = await IndexedDBConnector.create(schema)
```

Comment:
using `IndexedDBConnector` provided by unirep so that userState is stored locally.